### ISSUE TITLE
@at-root problem

### ISSUE THREAD

**Issue by khantry** [leafo/scssphp#577]
I have problem to compile @at-root with "&" code SCSS

sample: 
```
.class-name {
	color: blue;
	@at-root {
		.parent & {
			color: red;
		}
	}
}
```

and scssphp will result:
```
.class-name {color: blue}
.parent {color: red}
```

correct result:
```
.class-name {color: blue}
.parent .class-name {color: red}
```

**Comment by niaccurshi** [comment]
This has been fixed within #593

**Comment by Cerdic** [comment]
I can confirm this is no more an issue
